app/controllers/authenticate_controller.rb

```diff
@@ -72,14 +72,17 @@ def authenticate(input = authenticator_input)
       authenticators: installed_authenticators,
       enabled_authenticators: Authentication::InstalledAuthenticators.enabled_authenticators_str(ENV)
     )
-    content_type = :json
-    if encoded_response?
-      logger.debug(LogMessages::Authentication::EncodedJWTResponse.new)
-      content_type = :plain
-      authn_token = ::Base64.strict_encode64(authn_token.to_json)
-      response.set_header("Content-Encoding", "base64")
-    end
-    render(content_type => authn_token)
```

Comment:
why did you remove this?

Reply:
authenticate and authenticate_jwt use this same bock of code so i make a function (render_authn_token) out of it to avoid duplication of code